wpigui/src/main/native/include/portable-file-dialogs.h

```diff
@@ -123,6 +123,61 @@ namespace internal
 // Process wait timeout, in milliseconds
 constexpr int default_wait_timeout = 20;
 
+class platform
+{
+public:
+#if _WIN32
```

Comment:
This won't work in the header, it needs to be in the .cpp file (where it was deleted from).